ansible/ansible.cfg

```diff
@@ -5,6 +5,10 @@ nocows=1
 vault_password_file = ./gpg/get-vault-pass.sh
 
 gathering = smart
+fact_caching = jsonfile
+fact_caching_connection = ./facts
```

Comment:
Could we make that `.cache/facts` or something?
A dotted name will avoid having it polute the output of `ls`, and we might have more things we want to cache in the future (like OpenWRT imagebuilders, ...)